docs-src/modules/integrating/pages/integrating-with-feature-stores.adoc

```diff
@@ -38,7 +38,7 @@ Cassandra is a open source NoSQL distributed database that provides
 linear scalability and low-latency query results.
 
 Feature values can be written to Cassandra in real-time via Pulsar.
-First, link:Apache%20Pulsar[materialize features into Pulsar].
+First, <<apache-pulsar,materialize features into Pulsar>>.
```

Comment:
Will that link work? I thought it had to be a `x-ref` of some kind?

Reply:
tested locally. The `<<,>>` is another form of the xref macro

Reply:
https://docs.asciidoctor.org/asciidoc/latest/macros/xref/#internal-cross-references